src/data/storage-finder/service-list.json

```diff
@@ -0,0 +1,1326 @@
+[
+  {
+    "id": "9",
+    "title": "Amazon S3",
+    "facet_matches": [
+      "5",
+      "6",
+      "7",
+      "28",
+      "29",
+      "33",
+      "32",
+      "35",
+      "34",
+      "36",
+      "38",
+      "61",
+      "39",
+      "40",
+      "41",
+      "49",
+      "50",
+      "48",
+      "54",
+      "60",
+      "58",
+      "57",
+      "56",
+      "55",
+      "26",
+      "23",
+      "22",
+      "24",
+      "25",
+      "11",
+      "10",
+      "13",
+      "14",
+      "18",
+      "20",
+      "19"
+    ],
+    "summary": null,
+    "field_data": {
+      "field_alumni_access": {
+        "value": "\u003Cp\u003ENot Available\u003C\/p\u003E",
+        "label": "Alumni Access",
+        "weight": 10
+      },
+      "field_backup": {
+        "value": "\u003Cp\u003EAvailable for additional cost.\u003C\/p\u003E",
+        "label": "Backup",
+        "weight": 11
+      },
+      "field_eligibility": {
+        "value": "\u003Cp\u003EStaff and Faculty.\u003C\/p\u003E",
+        "label": "Eligibility",
+        "weight": 7
+      },
+      "field_limitations": {
+        "value": "\u003Cp\u003EAWS S3 is a chargeback service and an NYU chartfield is required for chargeback.\u003Cbr\u003EMax individual file upload size is 5TB.\u003C\/p\u003E\u003Cp\u003E\u0026nbsp;\u003C\/p\u003E",
+        "label": "Limitations",
+        "weight": 5
+      },
+      "field_links": {
+        "value": "\u003Cp\u003E\u003Ca href=\u0022https:\/\/www.nyu.edu\/servicelink\/KB0020307\u0022\u003EAmazon S3\u003C\/a\u003E\u003C\/p\u003E",
+        "label": "Links",
+        "weight": 2
+      },
+      "field_permission_settings": {
+        "value": "\u003Cp dir=\u0022ltr\u0022\u003ERestricted access by individual or group.\u003C\/p\u003E",
+        "label": "Permission Settings",
+        "weight": 6
+      },
+      "field_storable_files": {
+        "value": "\u003Cp\u003EHigh, Moderate, \u0026amp; Low Risk.\u003Cbr\u003E\u0026nbsp;\u003C\/p\u003E",
+        "label": "Storable Files",
+        "weight": 3
+      },
+      "field_synchronous_access": {
+        "value": "\u003Cp\u003ENot Available\u003C\/p\u003E",
+        "label": "Synchronous Access",
+        "weight": 9
+      },
+      "field_use_case": {
+        "value": "\u003Cp\u003EArchive and long-term, high-security storage of large datasets.\u003C\/p\u003E",
+        "label": "Use Case",
+        "weight": 4
+      }
+    }
+  },
+  {
+    "id": "15",
+    "title": "Distributed storage on the HSRN",
```

Comment:
If this is Ceph then most of the data is incorrect

Reply:
Do you know who manages the service running on `http://35.206.69.67`?